Total Instances: 82

Total Correct: 45
Total Runtime Failed: 31
Total Compilation Failed: 2
Total Test Failed: 4
Total Infinite Loop: 0

Accuracy: 54.87804878048781
Runtime Rate: 37.80487804878049
Compilation Rate: 2.4390243902439024
Test Failed Rate: 4.878048780487805
Infinite Loop Rate: 0.0

=================================================================================================
Failed Test Files: ['codeforces_672_A.py', 'codeforces_58_B.py', 'codeforces_622_A.py', 'atcoder_ARC062_B.py'] 
Failed Test Details: ['Filename: codeforces_672_A.py Actual: 3 Generated: 2\n', 'Filename: atcoder_ARC062_B.py Actual: 0\n Generated: -1\n', 'Filename: codeforces_622_A.py Actual: 311 Generated: 311.0\n', 'Filename: codeforces_58_B.py Actual: 733035 \n244345 \n48869 \n1  Generated: 733035 244345 48869 1 '] 
=================================================================================================
Runtime Error Files: ['codeforces_544_B.py', 'atcoder_ABC149_B.py', 'atcoder_ABC153_A.py', 'atcoder_ABC170_A.py', 'codeforces_379_A.py', 'codeforces_189_A.py', 'atcoder_ABC139_B.py', 'codeforces_546_A.py', 'codeforces_92_A.py', 'atcoder_ABC042_A.py', 'atcoder_ABC124_A.py', 'atcoder_AGC007_C.py', 'codeforces_651_A.py', 'codeforces_369_B.py', 'atcoder_ABC070_B.py', 'atcoder_ABC169_C.py', 'atcoder_ABC114_C.py', 'atcoder_ABC143_A.py', 'codeforces_86_A.py', 'codeforces_171_A.py', 'atcoder_ABC168_C.py', 'atcoder_ABC051_A.py', 'atcoder_AGC002_A.py', 'atcoder_ABC125_A.py', 'codeforces_30_A.py', 'codeforces_678_A.py', 'codeforces_96_B.py', 'codeforces_190_A.py', 'atcoder_ABC127_B.py', 'codeforces_669_A.py', 'atcoder_ABC158_B.py'] 
Runtime Error Details: ['Filename: codeforces_669_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_669_A.py", line 39, in <module>\n    codeforces_669_A()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_669_A.py", line 23, in codeforces_669_A\n    n = fr.nextInt()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_669_A.py", line 16, in nextInt\n    return int(self.next())\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_669_A.py", line 9, in next\n    self.st, self.st = self.st.split(\' \', 1)\nValueError: not enough values to unpack (expected 2, got 1)\n', 'Filename: atcoder_ABC124_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC124_A.py", line 11, in <module>\n    print(solve())\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC124_A.py", line 2, in solve\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'6 6\'\n', 'Filename: codeforces_544_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_544_B.py", line 24, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_544_B.py", line 19, in main\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'100 10000\'\n', 'Filename: atcoder_ABC169_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC169_C.py", line 3, in <module>\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'814793613347395 1.99\'\n', 'Filename: atcoder_ABC153_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC153_A.py", line 14, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC153_A.py", line 2, in main\n    monster_hp = int(input())\nValueError: invalid literal for int() with base 10: \'10000 1\'\n', 'Filename: codeforces_96_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_96_B.py", line 22, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_96_B.py", line 17, in main\n    value = int(s, 2)\nValueError: invalid literal for int() with base 2: \'74\'\n', 'Filename: codeforces_678_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_678_A.py", line 8, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_678_A.py", line 2, in main\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'999999999 1\'\n', 'Filename: atcoder_AGC002_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_AGC002_A.py", line 13, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_AGC002_A.py", line 2, in main\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'-1000000000 -1\'\n', 'Filename: atcoder_ABC143_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC143_A.py", line 10, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC143_A.py", line 4, in main\n    A = int(sys.stdin.readline())\nValueError: invalid literal for int() with base 10: \'75 25\\n\'\n', 'Filename: codeforces_546_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_546_A.py", line 20, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_546_A.py", line 5, in main\n    o = int(input())\nValueError: invalid literal for int() with base 10: \'1000 1000000000 1000\'\n', 'Filename: codeforces_86_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_86_A.py", line 16, in <module>\n    l = int(input())\nValueError: invalid literal for int() with base 10: \'999999999 999999999\'\n', 'Filename: atcoder_ABC158_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC158_B.py", line 15, in <module>\n    solver.run()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC158_B.py", line 3, in run\n    N = int(input())\nValueError: invalid literal for int() with base 10: \'501719768609305159 370614275896614890 131105492712690269\'\n', 'Filename: codeforces_651_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_651_A.py", line 22, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_651_A.py", line 2, in main\n    num1 = int(input())\nValueError: invalid literal for int() with base 10: \'100 25\'\n', 'Filename: codeforces_171_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_171_A.py", line 9, in <module>\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'35005211 521595368\'\n', 'Filename: atcoder_ABC070_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC070_B.py", line 22, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC070_B.py", line 2, in main\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'0 50 50 100\'\n', 'Filename: codeforces_30_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_30_A.py", line 16, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_30_A.py", line 4, in main\n    A = int(sys.stdin.readline())\nValueError: invalid literal for int() with base 10: \'7 896 7\\n\'\n', 'Filename: atcoder_ABC125_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC125_A.py", line 1, in <module>\n    n, k, m, l, a, b, c = map(int, input().split())\nValueError: not enough values to unpack (expected 7, got 3)\n', 'Filename: atcoder_AGC007_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_AGC007_C.py", line 20, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_AGC007_C.py", line 4, in main\n    N = float(input())\nValueError: could not convert string to float: \'1000 100 100\'\n', 'Filename: atcoder_ABC139_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC139_B.py", line 14, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC139_B.py", line 2, in main\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'8 8\'\n', 'Filename: codeforces_190_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_190_A.py", line 21, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_190_A.py", line 2, in main\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'59 96\'\n', 'Filename: atcoder_ABC170_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC170_A.py", line 14, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC170_A.py", line 5, in main\n    x = int(input())\nValueError: invalid literal for int() with base 10: \'1 2 3 4 0\'\n', 'Filename: atcoder_ABC042_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC042_A.py", line 23, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC042_A.py", line 2, in main\n    num_1 = int(input())\nValueError: invalid literal for int() with base 10: \'5 5 7\'\n', 'Filename: codeforces_92_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_92_A.py", line 20, in <module>\n    codeforces_92_A()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_92_A.py", line 2, in codeforces_92_A\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'24 7440\'\n', 'Filename: codeforces_379_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_379_A.py", line 13, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_379_A.py", line 2, in main\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'80 970\'\n', 'Filename: codeforces_189_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_189_A.py", line 19, in <module>\n    codeforces_189_A.main(sys.argv)\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_189_A.py", line 6, in main\n    n = int(inp())\nValueError: invalid literal for int() with base 10: \'29 27 18 2\\n\'\n', 'Filename: atcoder_ABC149_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC149_B.py", line 15, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC149_B.py", line 2, in main\n    A = int(input())\nValueError: invalid literal for int() with base 10: \'1000000000000 1000000000000 1000000000000\'\n', 'Filename: atcoder_ABC168_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC168_C.py", line 3, in <module>\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'1 2 9 49\'\n', 'Filename: atcoder_ABC127_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC127_B.py", line 1, in <module>\n    r = int(input())\nValueError: invalid literal for int() with base 10: \'2 1 2\'\n', 'Filename: atcoder_ABC051_A.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC051_A.py", line 8, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC051_A.py", line 4, in main\n    a, b, c = re.split(\',\', input())\nValueError: too many values to unpack (expected 3)\n', 'Filename: atcoder_ABC114_C.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 9, in <module>\n    ans += f(3 * (10 ** (len(S) - 1)), len(S) - 2)\nNameError: name \'f\' is not defined\n', 'Filename: codeforces_369_B.py Error_type: Traceback (most recent call last):\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_369_B.py", line 30, in <module>\n    main()\n  File "/home/changshu/CodeMind/Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python-sanitized/codeforces_369_B.py", line 20, in main\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'1 1 1 1 1 1\'\n'] 
=================================================================================================
Compilation Error Files: ['atcoder_AGC006_B.py', 'codeforces_678_B.py'] 
=================================================================================================
Infinite Loop Files: [] 
=================================================================================================
